src/Ast/PhpClassMember.php

```diff
@@ -23,6 +23,7 @@ abstract class PhpClassMember extends AST
 {
     use HasPhpDoc;
     use HasAccess;
+    use HasType;
```

Comment:
I would suggest moving this to just `PhpProperty`, as the other three classes which extend this class (`PhpConstant`, `PhpComment`, and `PhpMethod`) don't support types in the same way:

 - Constants: defining types are invalid (since they're implicit from the constant value)
 - Comments: don't have types
 - Methods: types are already defined by parameters return types

And we probably want to just get rid of the trait all together, since traits are meant for reusing code, and we will no longer be doing that.

Reply:
Great comment. Makes sense, let me change it.